<?xml version="1.0" ?>
<launch>
	
	<include file="$(find stdr_robot)/launch/robot_manager.launch" />
	
	<node type="stdr_server_node" pkg="stdr_server" name="stdr_server" output="screen" args="$(find sorting_robot)/stdr_data/maps/small_map_binary_grid.yaml"/>

	<node pkg="tf" type="static_transform_publisher" name="world2map" args="0 0 0 0 0 0  world map 100" />
	
	<include file="$(find stdr_gui)/launch/stdr_gui.launch"/>
	
	<!-- <node pkg="stdr_robot" type="robot_handler" name="$(anon robot_spawn)" args="add $(find stdr_test)/robots/pandora_robot.yaml 10 15 1.5708" /> -->

</launch>
